dependencies/src/test/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphIntegrationTest.java

```diff
@@ -71,7 +71,11 @@ public void testFindUpdates()
         "com.google.auth:google-auth-library-oauth2-http:0.9.1 needs to "
         + "upgrade com.google.http-client:google-http-client:1.19.0 to 1.23.0",
         "com.google.http-client:google-http-client-jackson2:1.19.0 needs to "
```

Comment:
Updates to these tests are due to the changes to `resolveCompileTimeDependencies` to include `provided` scope.